Create good for business blog (#58)
inclusivenaming/website: content/blog/_posts/inclusion-good-for-business.md

---
title: "Making Inclusion Part of Your Business is Better for Business. Period."

> date: 2021-05-14

Author: ```suggestion
date: 2021-5-26
```

> date: 2021-05-26

Reviewer: ```suggestion
date: 2021-06-01
```

slug: inclusive-good-for-business

> author: Wendy Renstchler

Reviewer 2: Wendy

> author: Wendy Renstchler

Reviewer 2: Wendy Rentschler, Shahed Alqadi

> author: Wendy Renstchler

Author: ```suggestion
author: Wendy Renstchler, Shahed Alqadi
```

---
title: "Making Inclusion Part of Your Business is Better for Business. Period."
date: 2021-05-14
slug: inclusive-good-for-business
author: Wendy Renstchler
---

> As we mark the first Black History Month following the events of 2020, attention is being paid to how companies are responding to their learnings from the Black Lives Matter movement and putting them into practice. And more companies have recognized that one month a year doesn’t come close to being enough. The enormity of social justice inequalities coming to light last year has been a catalyst for many businesses to look within themselves at how they do business—as an organization, as an employer, and as a member of the community and the world.

Author: ```suggestion
As we mark the first Black History month following the events of 2020, attention is being paid to how companies are responding to their learnings from the Black Lives Matter movement and putting them into practice. And more companies have recognized that one month a year doesn’t come close to being enough. The enormity of social justice inequalities coming to light last year has been a catalyst for many businesses to look within themselves at how they do business—as an organization, as an employer, and as a member of the community and the world.
```

> As we mark the first Black History Month following the events of 2020, attention is being paid to how companies are responding to their learnings from the Black Lives Matter movement and putting them into practice. And more companies have recognized that one month a year doesn’t come close to being enough. The enormity of social justice inequalities coming to light last year has been a catalyst for many businesses to look within themselves at how they do business—as an organization, as an employer, and as a member of the community and the world.

Reviewer 2: ~~As we mark the first Black History Month following the events of 2020, attention is being paid to how companies are responding to their learnings from the Black Lives Matter movement and putting them into practice.  And more companies have recognized that one month a year doesn’t come close to being enough.~~ 
As we continue to advance into the age of technology, attention is being paid to how companies are responding to their learnings from the various social justice initiatives surrounding us.

> As we mark the first Black History Month following the events of 2020, attention is being paid to how companies are responding to their learnings from the Black Lives Matter movement and putting them into practice. And more companies have recognized that one month a year doesn’t come close to being enough. The enormity of social justice inequalities coming to light last year has been a catalyst for many businesses to look within themselves at how they do business—as an organization, as an employer, and as a member of the community and the world.

Author: ```suggestion
As we continue to advance into the age of technology, attention is being paid to how companies are responding to their learnings from the various social justice initiatives surrounding us. The enormity of social justice inequalities coming to light last year has been a catalyst for many businesses to look within themselves at how they do business—as an organization, as an employer, and as a member of the community and the world.
```

The recent _[Fortune/Deloitte CEO Study](https://www2.deloitte.com/us/en/pages/chief-executive-officer/articles/ceo-survey-gauging-priorities.html)_ found that 96 percent of CEOs now consider diversity, equity, and inclusion (that’s the DEI you’re starting to see more often in the corporate vernacular) to be a strategic priority, integral to building a culture that’s better reflective of the world in which we live. And it acknowledges and embraces the uniqueness that every single person brings to a company’s vision. Together, we are better, and that’s better for business.

## Watch your language

> One way to do this from the outset is to recognize the [importance of language](https://www.bmc.com/blogs/2021-remote-work-considerations/) and how we use it in speech and the written word. As the world has changed, and awareness along with it, once-standard terminology is being retired and replaced with more inclusive language. So, what do we mean by that?

Reviewer 2: One way to do this from the outset is to recognize the ~[importance of language](https://www.bmc.com/blogs/2021-remote-work-considerations/)~ importance of language and how we use it in speech and the written word. As the world has changed, and awareness along with it, once-standard terminology is being retired and replaced with more inclusive language. So, what do we mean by that?

> One way to do this from the outset is to recognize the [importance of language](https://www.bmc.com/blogs/2021-remote-work-considerations/) and how we use it in speech and the written word. As the world has changed, and awareness along with it, once-standard terminology is being retired and replaced with more inclusive language. So, what do we mean by that?

Author: ```suggestion
One way to do this from the outset is to recognize the importance of language and how we use it in speech and the written word. As the world has changed, and awareness along with it, once-standard terminology is being retired and replaced with more inclusive language. So, what do we mean by that?
```

> One way to do this from the outset is to recognize the [importance of language](https://www.bmc.com/blogs/2021-remote-work-considerations/) and how we use it in speech and the written word. As the world has changed, and awareness along with it, once-standard terminology is being retired and replaced with more inclusive language. So, what do we mean by that?

Author: ```suggestion
One way to do this from the outset is to recognize the importance of language and how we use it in speech and the written word. As the world has changed, and awareness along with it, once-standard terminology is being retired and replaced with more inclusive language. So, what do we mean by that?
```

The [Inclusive Naming Initiative](https://inclusivenaming.org/) is a business collective tasked with defining processes and tools to remove harmful language and replace it with neutral terms. BMC is [proud](https://www.bmc.com/corporate/corporate-responsibility-impact.html) to be one of its member organizations.

DEI initiatives also strive to address issues of gender, gender identity, and gender discrimination, and there are new rules for that, too. For example, new social norms around gender identity have created an awareness and ownership of pronouns. It’s rapidly becoming a part of the social discourse, and social media, to include mention of “he/him,” “she/her,” or “they/them” in how you identify and wish to be referred to in the third person.

> [Here](https://s7280.pcdn.co/wp-content/uploads/2021/02/Inclusive-language-list.pdf) is a basic guide to problematic terms you might use every day without being conscious of their impact, and alternative options. BMC is addressing this with our own employees and partners through a style guide [resource](https://s7280.pcdn.co/wp-content/uploads/2021/02/Language-compressed-3.pdf) on how to use inclusive language.

Reviewer 2: ~[Here](https://s7280.pcdn.co/wp-content/uploads/2021/02/Inclusive-language-list.pdf)~ [Here] (https://inclusivenaming.org/language/word-list) is a basic guide published by the Inclusive Naming Initiative listing some problematic terms you might use every day without being conscious of their impact, and alternative options. BMC is addressing this with our own employees and partners through a style guide [resource](https://s7280.pcdn.co/wp-content/uploads/2021/02/Language-compressed-3.pdf) on how to use inclusive language.

> [Here](https://s7280.pcdn.co/wp-content/uploads/2021/02/Inclusive-language-list.pdf) is a basic guide to problematic terms you might use every day without being conscious of their impact, and alternative options. BMC is addressing this with our own employees and partners through a style guide [resource](https://s7280.pcdn.co/wp-content/uploads/2021/02/Language-compressed-3.pdf) on how to use inclusive language.

Author: ```suggestion
[Here](https://inclusivenaming.org/language/word-list) is a basic guide to problematic terms you might use every day without being conscious of their impact, and alternative options. BMC is addressing this with our own employees and partners through a style guide [resource](https://s7280.pcdn.co/wp-content/uploads/2021/02/Language-compressed-3.pdf) on how to use inclusive language.
```

AI, machine learning, and advanced analytics are also helping companies keep a better eye on how their employees feel about their workplace and their jobs through employee surveys and well-being assessments that gauge satisfaction levels, and algorithmic matching to advancement opportunities.

That said, there are still gaps to fill between action and perception. A recent multi-year [PwC study](https://www.pwc.com/gx/en/services/people-organisation/global-diversity-and-inclusion-survey.html) found that while 63 percent of business leaders believe they’re doing a good job communicating about employee and leadership diversity, only 42 percent of employees agree with them—and 33 percent feel that diversity is a barrier to their progression within the organization.

> At the end of the day, implementing DEI creates equitable opportunities for everyone to succeed, and that’s just good business, with both social and financial returns on investment. _[The Wall Street Journal](https://www.wsj.com/articles/the-business-case-for-more-diversity-11572091200)_’s research of DEI across the S&P 500 discovered that the 20 most diverse companies among those it surveyed have better operating results on average than the lowest-scoring firms, and their shares typically outperform those of the least-diverse firms, proving that inclusion is not only the right thing to do but is also the business imperative of our time.

Reviewer 2: At the end of the day, implementing DEI and robust Inclusive Naming policies creates equitable opportunities for everyone to succeed, and that’s just good business, with both social and financial returns on investment. _[The Wall Street Journal](https://www.wsj.com/articles/the-business-case-for-more-diversity-11572091200)_’s research of DEI across the S&P 500 discovered that the 20 most diverse companies among those it surveyed have better operating results on average than the lowest-scoring firms, and their shares typically outperform those of the least-diverse firms, proving that inclusion is not only the right thing to do but is also the business imperative of our time.

> At the end of the day, implementing DEI creates equitable opportunities for everyone to succeed, and that’s just good business, with both social and financial returns on investment. _[The Wall Street Journal](https://www.wsj.com/articles/the-business-case-for-more-diversity-11572091200)_’s research of DEI across the S&P 500 discovered that the 20 most diverse companies among those it surveyed have better operating results on average than the lowest-scoring firms, and their shares typically outperform those of the least-diverse firms, proving that inclusion is not only the right thing to do but is also the business imperative of our time.

Author: ```suggestion
At the end of the day, implementing DEI and robust Inclusive Naming policies creates equitable opportunities for everyone to succeed, and that’s just good business, with both social and financial returns on investment. _[The Wall Street Journal](https://www.wsj.com/articles/the-business-case-for-more-diversity-11572091200)_’s research of DEI across the S&P 500 discovered that the 20 most diverse companies among those it surveyed have better operating results on average than the lowest-scoring firms, and their shares typically outperform those of the least-diverse firms, proving that inclusion is not only the right thing to do but is also the business imperative of our time.
```

> The strength of BMC comes from our people who live our values every day, which include, “Prioritize People” and “Do the Right Thing.” As part of living this commitment, we advocate for awareness, empathy, and curiosity to understand each other, as a business and a culture, because the [Autonomous Digital Enterprise](https://www.bmc.com/corporate/autonomous-digital-enterprise.html) includes everyone.

Reviewer 2: ~The strength of BMC comes from our people who live our values every day, which include, “Prioritize People” and “Do the Right Thing.” As part of living this commitment, we advocate for awareness, empathy, and curiosity to understand each other, as a business and a culture, because the [Autonomous Digital Enterprise](https://www.bmc.com/corporate/autonomous-digital-enterprise.html) includes everyone.~

> The strength of BMC comes from our people who live our values every day, which include, “Prioritize People” and “Do the Right Thing.” As part of living this commitment, we advocate for awareness, empathy, and curiosity to understand each other, as a business and a culture, because the [Autonomous Digital Enterprise](https://www.bmc.com/corporate/autonomous-digital-enterprise.html) includes everyone.

Author: ```suggestion
```
